backend/api/views.py

```diff
@@ -28,10 +28,12 @@
 import string
 import random
 import datetime
+import stripe
 from oauth2_provider.models import get_application_model, get_access_token_model
 
 Application = get_application_model()
 AccessToken = get_access_token_model()
+stripe.api_key = "sk_test_51Hu2LSG8eUBzuEBE83xKbP5GrcDJVnBclJ7P5u95qOCF33C3NjdHqLlR4ICvYIQNYeVknFYjeZUxGD9aRcXX1TnT00i227Z5Pv"
```

Comment:
Tested it and LGTM 👍🏽  Feel free to remove it once you're done.

Reply:
Can we add it to the secrets before I remove them. I feel like i'll forget